lib/fog/libvirt/models/compute/templates/ceph.conf

```diff
@@ -0,0 +1,5 @@
+monitor=192.168.100.50,192.168.100.51,192.168.100.52
+port=6789
+libvirt_ceph_pool=foreman-images
+auth_username=foreman
+auth_uuid=8e1050ae-a53c-4280-bc48-8abe0eebafb1
```

Comment:
Aren't those numbers specific to your enviroment? Maybe this file should be in `.gitignore`?